recipe/meta.yaml

```diff
@@ -38,4 +38,5 @@ about:
 
 extra:
   recipe-maintainers:
```

Comment:
Actually! while we're here... `python >= 3.6` is required for git-machete (type annotations, `f"...{...}..."` style strings etc.)

Reply:
Please feel free to open a PR via the standard conda-forge mechanism.
Fork the repo, add the version info on a branch and open a PR against this repo.
I'll walk you through the update process there.

Reply:
Okay! @amalota could you handle that?

Reply:
@PawelLipski Sure!

Reply:
Pls also mention the existence of this repo (and provide instruction for any maintenance that we might want to do here) in our own CONTRIBUTING.md so that this knowledge isn't lost